README.md

```diff
@@ -2,77 +2,80 @@
+- Update `./.ddev/web-build/custom.cron` with your required commands.
+  - Remove `#ddev-generated` to prevent DDEV from overriding the file.
+- Custom the cron service, if required, by updating `./.ddev/web-build/cron.conf`
+  - Remove `#ddev-generated` to prevent DDEV from overriding the file.
+- Restart DDEV to apply any changes:
 
-The `config.cron.yaml` is a simple setup of a trivial cron job within the DDEV web container. It writes a crontab file to configure the cron daemon.
+  ```shell
+  ddev restart
+  ```
 
-```yaml
-hooks:
-  post-start:
-    # This adds an every-minute cronjob for your user; it runs "date" and appends that
-    # to the "time.log" in your project root.
-    # You can just `ls -l time.log` or `tail -f time.log` to see it happening.
-    # The crontab can have more than one line for multiple jobs.
-    # `ddev exec crontab -l` will show you the current crontab configuration
-    - exec: printf "SHELL=/bin/bash\n* * * * * date >> /var/www/html/time.log\n" | crontab
-```
+## Implementation
 
-The default file configures a job to write the date to a log file `time.log` every minute.
-It is a simple arbitary example to show the service is working, and remind the user to change it to something more appropriate. You can add additional files into /etc/cron.d, or add additional lines to this one.
+Out of the box, this add-on writes the date to a log file `time.log` every minute.
+This serves as an example implementation, provides proof the service is working, and is the basis for tests.
 
 - If you need help figuring out the syntax of a cron job, see [crontab guru](https://crontab.guru/).
 - For the usage of `crontab` see [crontab man page](https://manpages.debian.org/buster/cron/crontab.1.en.html).
-- You can experiment with the `crontab` command inside the container by `ddev ssh` and then `crontab -e` for example, or use `ddev exec crontab -e`.
+- Experiment with the `crontab` command inside the container by `ddev ssh` and then `crontab -e` for example, or use `ddev exec crontab -e`.
 - If you want the cron to run on your local time instead of UTC, make sure to set `timezone` in your `.ddev/config.yaml`.
 - Make sure that when you have tried manually executing the command you want to run inside the container and that it gets the expected results.
 - If you are running a CMS command that requires access to the database, set the environment variable `IS_DDEV_PROJECT=true`
 
 ## Examples
 
-**TYPO3 scheduler**: A cron to add on to the example and then run the TYPO3 scheduler every minute might be:
+### TYPO3 scheduler
 
-```yaml
-  - exec: printf "SHELL=/bin/bash\n* * * * * date |& tee -a /var/www/html/time.log\n* * * * * IS_DDEV_PROJECT=true /var/www/html/vendor/bin/typo3 scheduler:run -vv |& tee -a /var/www/html/scheduler-log.txt\n" | crontab
+A cron to add on to the example and then run the TYPO3 scheduler every minute might be:
```

Comment:
Add words to explain where they would put this cron. They probably skimmed over above. Best to put in an extra file if possible.

Reply:
Update example sections